CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
 
   ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
 
+- Update padding of govuk-main-wrapper
```

Comment:
I'm not 100% sure this is a breaking change, but we have not really figured out if rhythm changes should count as a breaking change...